### ISSUE TITLE
waveorder changes matplotlib backend at import

### ISSUE THREAD

**Issue by ieivanov** [mehta-lab/waveorder#133]
The waveorder `__init__.py` file imports all library modules, even if they are not used. This includes `from .visual import *`, which changes the default `matplotlib` backend, in my case to `QtAgg`, even if function from the `visual` module are not used, which can create problems with other libraries. Can we remove the `from .visual import *` line, or even the entire contents of this file? I think this is not a good practice in general.

**Comment by talonchandler** [comment]
Yes 100%! I've been looking for an excuse to do this errand for a while. 

I'll open a draft PR right away, and I'll make it a full PR if/when the tests pass.

**Comment by ieivanov** [comment]
Thank you!
